pkg/oc/admin/diagnostics/pod.go

```diff
@@ -89,37 +80,26 @@ func (o *PodDiagnosticsOptions) Complete(args []string) error {
 	return nil
 }
 
-// BuildAndRunDiagnostics builds diagnostics based on the options and executes them, returning a summary.
-func (o PodDiagnosticsOptions) BuildAndRunDiagnostics() (bool, error, int, int) {
-	failed := false
-	errors := []error{}
-	diagnostics := []types.Diagnostic{}
+// BuildAndRunDiagnostics builds diagnostics based on the options and executes them, returning fatal error(s) only.
+func (o PodDiagnosticsOptions) BuildAndRunDiagnostics() error {
```

Comment:
consider renaming to just `RunDiagnostics`

Reply:
Even though it's ultimately calling something else named RunDiagnostics? After building them...

Reply:
Yeah, it would follow the pattern set in other cmds closer, and would be easier to think of this func as a wrapper for util.RunDiagnostics